cmd/show/show.go

```diff
@@ -0,0 +1,231 @@
+	PushedAt          time.Time `json:"pushed_at"`
+	DefaultBranch     string    `json:"default_branch"`
+	NodeID            string    `json:"node_id"`
+	GitURL            string    `json:"git_url"`
+	SSHURL            string    `json:"ssh_url"`
+	CloneURL          string    `json:"clone_url"`
+	SVNURL            string    `json:"svn_url"`
+	MirrorURL         string    `json:"mirror_url"`
+	Name              string    `json:"name"`
+	FullName          string    `json:"full_name"`
+	Description       string    `json:"description"`
+	Language          string    `json:"language"`
+	License           string    `json:"license"`
+	URL               string    `json:"url"`
+	Homepage          string    `json:"homepage"`
+	Topics            []string  `json:"topics"`
+	ContributionCount int       `json:"contributionsCount"`
+	VotesCount        int       `json:"votesCount"`
+	SubmissionCount   int       `json:"submissionsCount"`
+	StargazersCount   int       `json:"stargazersCount"`
+	StarsCount        int       `json:"starsCount"`
+}
+
+func run(opts *Options) error {
+
+	err := validateShowQuery(opts)
+	if err != nil {
+		return err
+	}
+
+	var client = &http.Client{Timeout: 10 * time.Second}
+
+	// retrieve repository info
+	resp, err := client.Get(fmt.Sprintf("https://api.opensauced.pizza/v1/repos/%s", opts.RepoName))
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	var repoInfo reposResponse
+	err = json.NewDecoder(resp.Body).Decode(&repoInfo)
+	if err != nil {
+		return err
+	}
+
+	if repoInfo.ID == 0 {
+		return errors.New("could not find repository")
+	}
+
+	newContributors, err := FetchNewContributors(opts, repoInfo.Name, repoInfo.ID)
+	if err != nil {
+		return err
+	}
+
+	churnContributors, err := FetchNewContributors(opts, repoInfo.Name, repoInfo.ID)
+	if err != nil {
+		return err
+	}
+
+	// Load the pizza TUI
+	pizzaTUI(newContributors.Contributors, churnContributors.Contributors)
+
+	return nil
+}
+
+func validateShowQuery(opts *Options) error {
+	if opts.RepoName == "" {
+		return errors.New("no repository was provided")
+	}
+
+	if len(strings.Split(opts.RepoName, "/")) != 2 {
+		return errors.New("wrong repository name format. Must be in the form owner/name, i.e(open-sauced/pizza)")
+	}
+	if opts.Limit < 1 {
+		return errors.New("--limit flag must be a positive integer value")
+	}
+	if opts.Range < 1 {
+		return errors.New("--range flag must be a positive integer value")
+	}
+
+	if opts.Page < 1 {
+		return errors.New("--page flag must be a positive integer value")
+	}
+
+	return nil
+}
+
+// FetchNewContributors: Returns all the new contributors
+func FetchNewContributors(opts *Options, repoName string, repoID int) (showRespose, error) {
+	var client = &http.Client{Timeout: 10 * time.Second}
+	var newContributors showRespose
+
+	resp, err := client.Get(fmt.Sprintf("https://api.opensauced.pizza/v1/contributors/insights/new?page=%d&limit=%d&range=%d&topic=javascript&repo=%s&repoIds=%d",
```

Comment:
The `topic` and `repo` params aren't needed here. Fixed in https://github.com/open-sauced/api/pull/265

```suggestion
	resp, err := client.Get(fmt.Sprintf("https://api.opensauced.pizza/v1/contributors/insights/new?page=%d&limit=%d&range=%d&repoIds=%d",
```